src/introduction.md

```diff
@@ -77,6 +77,10 @@ TypeScript transpiles down to JavaScript, a dynamic just-in-time compiled langua
 It likely won't. While TypeScript adds typings to JavaScript, it is a superset after all and can describe many of JavaScript's dynamic features, not all of which are feasible to support in ahead-of-time compilation. Yet, sufficiently strict TypeScript code can often be made compatible with the AssemblyScript compiler with little effort.
 :::
 
+::: tip Can I use any third party npm package like web.js or crypto.js?
+No. Even if the npm package has TypeScript sources that are fully compatible with AssemblyScript, bundlers / transpilers will likely compile it into regular JavaScript before publishing to npm. However, you can use packages from npm that are built specifically for AssemblyScript. Some of them can be found at [Built with AssemblyScript](./built-with-assemblyscript.md). Furthermore, you can always import the necessary JavaScript methods from the host (browser or node.js) via [host bindings](./compiler.html#host-bindings).
```

Comment:
I think that "no" is not quite correct, since the recommended way is to then bind to them. Not sure how hard it will be to explain all this in a few sentences in the FAQ, though.

Basic points:
* AssemblyScript module with source code available: can `import`
* Any other module: Requires either using host bindings (likely with `declare`s on the AS side) or something manual

So I guess the answer is "it depends" :)

Reply:
Depends on host? Due to not all hosts can run JS and bind JS. For example node.js & browser can, but wasmtime or wasmer can't